Lib/elements/input_field.py

```diff
@@ -0,0 +1,32 @@
+"""Class for handling input field elements."""
+from base_element import BaseElement
+
+
+class InputField(BaseElement):
+    def __init__(self, driver, element_tuple, timeout=10):
```

Comment:
You can use construction of the base class instead of defining the same one here